spring-rabbit-ext/README.adoc

```diff
@@ -0,0 +1,5 @@
+= spring-rabbit-ext
+
+This project provides extensions to `spring-rabbit`.
+
+In particular, the `log4j2 Appender` is contained here, to avoid classpath conflicts with `spring-rabbit`.
```

Comment:
I read somewhere that you can't have both log4j and log4j2 on the classpath so I was concerned that adding it to the main project would break the log4j Appender tests. But we should try it I suppose; I think that discussion might have been about the log4j1 -> log4j2 bridge.